packages/react/src/ReactElement.js

```diff
@@ -46,6 +46,22 @@ function hasValidKey(config) {
   return config.key !== undefined;
 }
 
+function checkPropsForNull(props) {
+  let propName;
```

Comment:
Great update!

One last thing, I would move the definition for `propName` in to the loop as in `for (let propName in origProps)`.